cfme/infrastructure/pxe.py

```diff
@@ -1128,5 +1128,5 @@ def remove_all_pxe_servers():
     view = navigate_to(PXEServer, 'All')
     if view.entities.is_displayed:
         for entity in view.entities.rows():
-            entity[0].check()
+            entity[0].ensure_checked()
```

Comment:
Added tp PRT to cover this. cfme/tests/physical_infrastructure/ui/test_physical_server_list_buttons.py::test_manage_button
I was unable to get the test run to hit the updated code when running locally, but am hoping PRT might catch it. There were no pxe servers for me to delete when running locally.